tests/integration/test_model.py

```diff
@@ -334,3 +338,73 @@ def test_describe_model(c, training_df):
 
     with pytest.raises(RuntimeError):
         c.sql("DESCRIBE MODEL undefined_model")
+
+
+def test_export_model(c, training_df, tmpdir):
+    with pytest.raises(RuntimeError):
```

Comment:
Well done! You are testing the corner cases - really good!

Reply:
learned from you, Thanks ❤️